lib/internal/bootstrap_node.js

```diff
@@ -204,7 +204,7 @@
       // getter
       const get = util.deprecate(function() {
         return this;
-      }, `'${name}' is deprecated, use 'global'`);
+      }, `'${name}' is deprecated, use 'global'`, 'DEP0016');
```

Comment:
If the warning in `fs.js` is actually `DEP0016`, then this and whatever follows this have to be changed.